digiroad2-geo/src/main/scala/fi/liikennevirasto/digiroad2/linearasset/SpeedLimitFiller.scala

```diff
@@ -122,8 +123,16 @@ object SpeedLimitFiller {
     (limits, changeSet.copy(droppedAssetIds = changeSet.droppedAssetIds ++ limitsToDrop))
   }
 
+
+  private def isValidUnknownSpeedLimit(roadLink: RoadLink): Boolean = {
+    val roadLinkType = Seq(CycleOrPedestrianPath, PedestrianZone, TractorRoad, MotorwayServiceAccess, SpecialTransportWithoutGate, SpecialTransportWithGate, CableFerry)
+    val constructionType : Seq[ConstructionType] = Seq(UnderConstruction, Planned)
```

Comment:
We could put this inside the RoadLink case class in case we need to use it in some other place. (like it is done with the `def isCarTrafficRoad`)